all/experiments/multiagent_env_experiment.py

```diff
@@ -2,6 +2,7 @@
 import numpy as np
 from scipy import stats
 from .writer import ExperimentWriter
+from .writer import CometWriter
```

Comment:
You can use one line to import both of these, i.e. `from .writer import ExperimentWriter, CometWriter`